src/pyshark/tshark/tshark.py

```diff
@@ -82,13 +82,16 @@ def get_tshark_version(tshark_path=None):
     with open(os.devnull, "w") as null:
         version_output = subprocess.check_output(parameters, stderr=null).decode("ascii")
 
-    version_line = version_output.splitlines()[0]
+    version_lines = version_output.splitlines()
+    version_string = None
     pattern = r'.*\s(\d+\.\d+\.\d+).*'  # match " #.#.#" version pattern
```

Comment:
Optional regex to further narrow the version

```suggestion
    pattern = r'^TShark.*?\s(\d+\.\d+\.\d+).*'  # match " #.#.#" version pattern
```